user_sync/connector/directory_ldap.py

```diff
@@ -103,19 +107,28 @@ def __init__(self, caller_options):
                     'password': six.text_type(password)}
             logger.debug('Connecting to: %s - Authentication Method: NTLM using username: %s', options['host'],
                          options['username'])
+        elif auth_method == 'kerberos':
+            if(platform.system() == 'Windows'):
+                from .ldap3_extended.Connection import Connection
+                auth = {'authentication': ldap3.SASL, 'sasl_mechanism': ldap3.GSSAPI}
+                logger.debug('Connecting to: %s - Authentication Method: Kerberos', options['host'])
+            else:
+                raise AssertionException('Kerberos Authentication Method is not supported on this OS. Windows Only')
```

Comment:
Technically the ldap3 extended library will import kerberos since winkerberos will fail under linux but I got to look up how to join Linux OS to AD domain and setup kerberos client to test.

Reply:
@bhunut-adobe what flavor of Linux do you use? I will help you with client set up.